### PR TITLE
🧸 rht-labs/ubiquitous-journey 🧸

### PR DESCRIPTION
**Pull request by eformat**

### CODE REVIEW

File: tooling-values.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+sonatype-nexus:
+  enabled: true
+
 applications:
   - name: nexus
     enabled: true
-    source: https://github.com/tylerauerbeck/ubiquitous-journey
+    source: https://github.com/rht-labs/ubiquitous-journey
     source_path: charts/ubiquitous-journey
     source_ref: init-apps
```
Comment (springdo) [inline tooling-values.yaml]: probs should be master too - i have this change in my PR also though

File: tooling-values.yaml [hunk 2]
```diff
@@ -1,8 +1,11 @@
 
+sonatype-nexus:
+  enabled: true
+
```
Comment (springdo) [inline tooling-values.yaml]: Does this need to be here? think it's only needed in the apps/nexus/values?

Comment (springdo) [reply tooling-values.yaml]: @eformat  - i did a test of this, and it seems to work without this here in my argo on s43. Im getting sync issues cuz of ocp routes though

Comment (eformat) [reply tooling-values.yaml:4]: hmm .. nexus just does not enable for me without it ?
```
virt:~/git/ubiquitous-journey ⎇ master#1a40358$ helm2 template  -f tooling-values.yaml charts/ubiquitous-journey | grep nexus
  name: nexus
      releaseName: nexus
        - apps/nexus/values.yaml
virt:~/git/ubiquitous-journey ⎇ master#1a40358$ helm3 template  -f tooling-values.yaml charts/ubiquitous-journey | grep nexus
  name: nexus
      releaseName: nexus
        - apps/nexus/values.yaml
``` 

requirements.yaml -> condition: sonatype-nexus.enabled
values.yaml -> sonatype-nexus.enabled: false
charts/ubiquitous-journey/apps/nexus/values.yaml -> sonatype-nexus.enabled: true

so .. how is the requirements condition set ?

**Comment by eformat** [comment]
am going to close this - and merge your PR which has this change in there .. will go from there
https://github.com/rht-labs/ubiquitous-journey/pull/60

**Pull request closed without merging**